machine_management/index.adoc

```diff
@@ -0,0 +1,63 @@
+:context: overview-of-machine-management
+
+toc::[]
+
+You can use machine management to flexibly work with underlying infrastructure like Amazon Web Services (AWS), Azure, Google Cloud Platform (GCP), OpenStack, Red Hat Virtualization (RHV), and vSphere to manage the {product-title} cluster.
+You can control the cluster, and also perform auto-scaling, that is, scaling up and down the cluster based on specific workload policies.
+
+The {product-title} cluster can horizontally scale up and down, when the load increases or decreases.
+It is important to have a cluster that adapts to changing workloads.
+
+Machine management is implemented as a xref:../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Custom Resource Definition](CRD).
+A CRD object defines a new unique object `Kind` in the cluster and enables the Kubernetes API server to handle it's entire lifecycle.
+
+The Machine API Operator provisions the following resources:
+
+* MachineSet
+* Machine
+* Cluster Autoscaler
+* Machine Autoscaler
+* Machine Health Checks
+
+[discrete]
+== What you can do with machine sets
+
+As a cluster administrator you can:
+
+. Create a machine set on:
+** xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[AWS]
+** xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#creating-machineset-azure[Azure]
+** xref:../machine_management/creating_machinesets/creating-machineset-gcp.adoc#creating-machineset-gcp[GCP]
+** xref:../machine_management/creating_machinesets/creating-machineset-osp.adoc#creating-machineset-osp[OpenStack]
+** xref:../machine_management/creating_machinesets/creating-machineset-rhv.adoc#creating-machineset-rhv[RHV]
+** xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc#creating-machineset-vsphere[vSphere]
+
+. xref:../machine_management/manually-scaling-machineset.adoc#manually-scaling-machineset[Manually scale a machine set] by adding or removing a machine from the machine set.
+. xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Modify a machine set] through the MachineSet YAML configuration file.
+. xref:../machine_management/deleting-machine.adoc#deleting-machine[Delete] a machine.
+. xref:../machine_management/creating-infrastructure-machinesets.adoc#creating-infrastructure-machinesets[Create infrastructure machine sets].
+. Configure and deploy a xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[machine health check] to automatically fix damaged machines in a machine pool.
+
+[discrete]
+== Autoscaler
+
+The autoscaler ensures that your cluster is flexible to changing workloads.
+To xref:../machine_management/applying-autoscaling.adoc#applying-autoscaling[autoscale] your {product-title} cluster, you must first deploy a cluster autoscaler, and then deploy a machine autoscaler for each machine type.
```

Comment:
I think this should be `and then deploy a machine autoscaler for each **machine set**.`

Reply:
good catch! its resolved now.